okhttp/src/main/java/okhttp3/Handshake.java

```diff
@@ -49,6 +51,10 @@ public static Handshake get(SSLSession session) {
     if (cipherSuiteString == null) throw new IllegalStateException("cipherSuite == null");
     CipherSuite cipherSuite = CipherSuite.forJavaName(cipherSuiteString);
 
+    String tlsVersionString = session.getProtocol();
+    if (tlsVersionString == null) throw new IllegalStateException("tlsVersion == null");
+    TlsVersion tlsVersion = TlsVersion.forJavaName(tlsVersionString);
```

Comment:
Oh, derp. It's right here. I was missing this usage of it (and propagation to the ctor).